dluhc-component-library/src/main.tsx

```diff
@@ -1,24 +1,36 @@
 import { render } from "preact";
 import SiteSelectionForm from "./components/siteSelectionForm.tsx";
+import TimetablePage from "./components/timetablePage/TimetablePage.tsx";
 
 import "./index.css";
 
-type renderFunction = (options: {}, element: HTMLElement) => void;
+type Options = any;
```

Comment:
Some thought needs to go into this type so I have pulled it out to at least name it for now.